io/driver/module_unload_load.sh

```diff
@@ -16,7 +16,11 @@
 
 CONFIG_FILE="$AVOCADO_TEST_DATADIR"/config
 BUILT_IN_DRIVERS=`cat /lib/modules/$(uname -r)/modules.builtin |awk -F"/" '{print $NF}'|sed 's/\.ko//g'`
-DRIVERS=`find /lib/modules/$(uname -r)/ -name \*.ko | awk -F"/" '{print $NF}'|sed 's/\.ko//g'`
+if [[ $ONLY_IO == True ]]; then
```

Comment:
Also, in case the yaml parameter is empty or yaml file is not passed to test, the `if $ONLY_IO` will throw error, whereas comparison will not. So, this approach is preferred